contents/handbook/company/goal-setting.md

```diff
@@ -4,6 +4,25 @@ sidebar: Handbook
 showTitle: true
 ---
 
+## Operating Cadence
+
+PostHog works on a rolling 12 month outlook, referred to as medium-to-long term, for making tangible decisions that will affect what people are working on. This 12 month outlook is always being evaluated by the exec team who will always be assessing the challenges that the rest of the team will be accountable for solving. The aim here is that the exec team can set the direction and the wider team are able to autonomously solve the challenge of how to get there. 
```

Comment:
How frequently would you see us updating this rolling outlook? Should we be doing this on a specific cadence, like every 3 months?

Reply:
(I see you refer to this below, but I think we probably want a tiny bit more specificity here? e.g. we could tie it to board meetings?)